.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### Description
+
+<!--- Please include a summary of the change and which issue is fixed. --->
+<!--- Please also include relevant motivation and context. -->
+
+<!--- Add visual representation of the effect of the change when possible, e.g. before and after screenshots, code snippets, etc. --->
+
+Fixes #{issue}
+
+### How Has This Been Tested?
+
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce while reviewing your changes. -->
+
+### AI Disclosure
+<!--- Remove this section if your PR does not contain AI-generated content. --->
+- [ ] This PR contains AI-generated content.
+  - [ ] I have tested any AI-generated content in my PR.
+  - [ ] I take responsibility for any AI-generated content in my PR.
+
+### Checklist:
+<!--- Remove the non relevant items. --->
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works, or that the bug can not reappear.
+- [ ] I have verified that changes that would impact the JSON schema have been made in `schema/model.py`.
+- [ ] I have updated or added relevant documentation.
```

Comment:
I think we have a duplicate with
`I have made corresponding changes to the documentation`